DOCS/man/af.rst

```diff
@@ -189,6 +189,37 @@ Available filters are:
     ``window-size=<amount>``
         Length in milliseconds of the overlap-and-add window. (default: 12)
 
+``ffscaletempo[=[filter=]<filter_name>]``
```

Comment:
It'd be better to call this something generic like `lavfi-pitch-correction`. scaletempo refers to a specific class of pitch correction filters.